task/copy.go

```diff
@@ -185,7 +195,7 @@ func (t *CopyStreamTask) run() {
 	initTask := NewSegmentInit(t)
 	err := utils.ChooseDestinationStorageAsIndexSegmenter(initTask, t)
 	if err != nil {
-		t.TriggerFault(types.NewErrUnhandled(err))
+		t.TriggerFault(err)
```

Comment:
`err` has already been handled in  `ChooseDestinationStorageAsIndexSegmenter`, it will return a `StorageInsufficientAbility` err. So we do not need to wrap an `Unhandled` error here.